src/pages/createBook/components/InputUploadImg.tsx

```diff
@@ -35,26 +35,26 @@ export const InputUploadImg: FC<InputFileType> = ({onChange, errors, type, name,
 
     return (
         <>
```

Comment:
we don't need `<>` anymore, couse we have `div` wrapper

Reply:
No, we need them. They wrap two <div>